modules/module/BaseImageCaptionModel.py

```diff
@@ -1,4 +1,5 @@
 import os
+import pathlib
```

Comment:
Yeah, this is a great change. I always use `pathlib` since it's a million times better than `os.path.join` and other legacy garbage. :)

But should change it to `from pathlib import Path` to avoid the clunky usage.

https://docs.python.org/3/library/pathlib.html#basic-use